docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -161,6 +208,18 @@ event == "push" && target_branch == "v1.0.0"
 
 * The `appstudio.openshift.io/application` and `appstudio.openshift.io/component` labels must be adjusted to specify the right Application and Component respectively. Failing to do this will cause builds of the pipeline to be associated with the wrong application or component.
 
+* The value for the `output-image` parameter should be set to match the vale of the `spec.image.name` field of the `ImageRepository` resource that corresponds to the component the pipline would build. 
++
+If you template is setup in such a way that the repo branch name would appear in the image name, you can use the `target_branch` PaC variable like so:
++
+[source]
+----
+  - name: output-image
+    value: quay.io/redhat-user-workloads/my-tenant/my-app-{{target_branch}}/my-comp-{{target_branch}}:{{revision}}
+----
++
+Please note that the examples in this document to not allow for this setup because the image names conain the hyphenized version value while the branch names contain the non-hyphenized value.
```

Comment:
```suggestion
Please note that the examples in this document do not allow for this setup because the image names contain the hyphenated version value while the branch names contain the unhyphenated value.
```